packages/libs/pactjs-generator/src/contract/generation/FileContractDefinition.ts

```diff
@@ -13,10 +13,24 @@ export class FileContractDefinition implements IContractDefinition {
   private _logger: ILogger;
   private _raw: Output;
 
-  public constructor(filePath: PathLike, logger: ILogger = () => {}) {
+  public constructor(
```

Comment:
as the amount of params are increasing, do we want to consider destructuring an object over a list of arguments? Do we expect this constructor to grow in the near future?

Reply:
Actually disregarding whether we want to grow the list, if I now want to provide a logger, but no namespace, I would have to provide a `null` or `undefined` in between. I'd rather have a object destructured as api in such case